### PR TITLE
Fix translations workflow and use single transifex resource (no longer release-*)

### PR DESCRIPTION
**Pull request by suricactus**
Better delete the failing file, until the repo secrets are properly setup. This script is already properly working on TravisCI, so we are not missing anything.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-cd8b8fbafce048771cfd09dc4f83a3b07c1dd722-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-cd8b8fbafce048771cfd09dc4f83a3b07c1dd722-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-cd8b8fbafce048771cfd09dc4f83a3b07c1dd722-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-cd8b8fbafce048771cfd09dc4f83a3b07c1dd722-x86_64.apk)

**Comment by m-kuhn** [comment]
The secret `TX_TOKEN` is now available.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-52f4d2a0ac65e1a0d9d24de586f2c0caae4bf122-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-52f4d2a0ac65e1a0d9d24de586f2c0caae4bf122-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-52f4d2a0ac65e1a0d9d24de586f2c0caae4bf122-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-52f4d2a0ac65e1a0d9d24de586f2c0caae4bf122-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-ce07e0a201dd8b7a73e213d46896ced41f1e7c5f-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-ce07e0a201dd8b7a73e213d46896ced41f1e7c5f-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-ce07e0a201dd8b7a73e213d46896ced41f1e7c5f-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-ce07e0a201dd8b7a73e213d46896ced41f1e7c5f-x86_64.apk)

**Comment by suricactus** [comment]
@m-kuhn I also removed the automatic push to "release-*" resources on transifex.

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-3146558d9b55d73621a3f6886a7975e7075d4651-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-3146558d9b55d73621a3f6886a7975e7075d4651-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-3146558d9b55d73621a3f6886a7975e7075d4651-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-3146558d9b55d73621a3f6886a7975e7075d4651-x86_64.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-6a0c9d9b3785d7b8dedf12dcf241c8dc67d295ef-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-6a0c9d9b3785d7b8dedf12dcf241c8dc67d295ef-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-6a0c9d9b3785d7b8dedf12dcf241c8dc67d295ef-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-6a0c9d9b3785d7b8dedf12dcf241c8dc67d295ef-x86_64.apk)

File: .github/workflows/translations.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - gh_wf
```
Comment (m-kuhn) [inline .github/workflows/translations.yml:7]: Reminder: remove before merge

File: scripts/ci/update-translations.sh [hunk 2]
```diff
@@ -4,9 +4,8 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source ${DIR}/../version_number.sh
 
 lupdate QField.pro
+
+# release only if the branch is master, no longer transifex "release-*"
```
Comment (m-kuhn) [inline scripts/ci/update-translations.sh:8]:
```suggestion
# release only if the branch is master
```

No reason to comment historical considerations

Comment (suricactus) [reply scripts/ci/update-translations.sh]: clear

File: .github/workflows/translations.yml (hunk 1 above)
Comment (suricactus) [reply .github/workflows/translations.yml]: Done

**Comment by qfield-fairy** [comment]
Uploaded test apks for
 - [**armv7**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-c27b523b8c27b8acca5307d2fb6b2ae1f717025a-armv7.apk)
 - [**x86**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-c27b523b8c27b8acca5307d2fb6b2ae1f717025a-x86.apk)
 - [**arm64_v8a**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-c27b523b8c27b8acca5307d2fb6b2ae1f717025a-arm64_v8a.apk)
 - [**x86_64**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-1155-c27b523b8c27b8acca5307d2fb6b2ae1f717025a-x86_64.apk)

**Comment by m-kuhn** [comment]
@suricactus is this ready for a test on master?

**Comment by suricactus** [comment]
@m-kuhn waiting for approval, it seems all is working now

**Pull request merged**